components/MapViewScreen/index.tsx

```diff
@@ -74,9 +98,11 @@ export default function MapViewScreen({
         onFilterChange={handleFilterChange}
         selectedFilters={selectedFilters}
         setSelectedFilters={setSelectedFilters}
+        handleFilterButtonClick={handleFilterButtonClick}
+        clearFilters={clearFilters}
       />
-      <Map projects={projects} />
-      <ProjectsListingModal projects={filteredProjects} />
+      <Map projects={filteredProjects} />
```

Comment:
In this line, you are re-rending the map each time by passing in filteredProjects as a prop. Please change it back to projects. As I mentioned before, filtering the markers was Neha's sprint task, and the implementation she has will be making less API calls than your implementation.